Mathlib/GroupTheory/GroupAction/Group.lean

```diff
@@ -178,6 +180,36 @@ theorem smul_eq_iff_eq_inv_smul (g : α) {x y : β} : g • x = y ↔ x = g⁻¹
 
 end Group
 
+section Monoid
+
+variable [Monoid α] [MulAction α β]
+
+@[to_additive]
+instance : MulAction αˣ β where
+  one_smul _ := by
+    rw [one_smul]
+  mul_smul _ _ _ := by
+    rw [mul_smul]
+
+variable {c : α} {x y : β} [Invertible c]
+
+@[simp]
+theorem invOf_smul_smul : ⅟c • c • x = x := inv_smul_smul (unitOfInvertible c) _
+
+@[simp]
+theorem smul_invOf_smul : c • (⅟ c • x) = x := smul_inv_smul (unitOfInvertible c) _
+
+variable (c) in
+theorem smul_eq_iff  : c • x = c • y ↔ x = y := smul_left_cancel_iff (g := unitOfInvertible c)
```

Comment:
I don't think we should should make `Inverible` copies of lemmas which don't actually mention `invOf`

Reply:
I've removed these lemmas, since I don't need them, although I'm not sure if your objection is to the lemmas or the way they are named?